tutorials/getting_started/using_pretrained_models.md

```diff
@@ -0,0 +1,68 @@
+# Using Pretrained AllenNLP Models
+
+In this tutorial, we show how to use run the pretrained models in AllenNLP to make predictions.
+While this tutorial will specifically use the Named Entity Recognizer, but will apply to any of
+the models [available on our website](http://allennlp.org/models).
+
+## Making Predictions on the Command Line
+
+[The models page on the website](http://allennlp.org/models) lists all the models in AllenNLP,
+as well as examples for how to run the model on the command line.  For example, under the
+[Named Entity Recognition model](http://allennlp.org/models/#named-entity-recognition) there
+is a "Prediction" button that reveals the following example.
+
+```bash
+echo '{"sentence": "Did Uriah honestly think he could beat The Legend of Zelda in under three hours?"}' > ner-examples.jsonl
+python -m allennlp.run predict \
+    https://s3-us-west-2.amazonaws.com/allennlp/models/ner-model-2018.04.26.tar.gz \
+    ner-examples.jsonl
+```
+
+If no predictor is specified (as in the above example), then AllenNLP will use [the default
+predictor](https://github.com/allenai/allennlp/blob/ea2e431cf7672fd1d04bbd382141495bfbc021f7/allennlp/service/predictors/predictor.py#L12).
+You will need to create a custom predictor if you want to customize your input or output format.
+Here is an example of running the above example but specifying
+[the predictor](https://github.com/allenai/allennlp/blob/ea2e431cf7672fd1d04bbd382141495bfbc021f7/allennlp/service/predictors/sentence_tagger.py#L11)
+explicitly.
+
+```bash
+python -m allennlp.run predict \
+    https://s3-us-west-2.amazonaws.com/allennlp/models/ner-model-2018.04.26.tar.gz \
+    ner-examples.jsonl \
+    --predictor sentence-tagger
+```
+
+## Making Predictions Programatically
+
+You can also make productions from python, using AllenNLP as a library.  The following code example
+uses the default predictor (`sentence-tagger`) for the NER model.
+
+```python
+from allennlp.service.predictors import Predictor
+predictor = Predictor.from_path("https://s3-us-west-2.amazonaws.com/allennlp/models/ner-model-2018.04.26.tar.gz")
+results = predictor.predict("Did Uriah honestly think he could beat The Legend of Zelda in under three hours?")
```

Comment:
I would use named parameters here `sentence="Did Uriah..."` and call out explicitly that the parameters to `predict` depend on which predictor you're using